mlflow/sklearn/utils.py

```diff
@@ -299,23 +299,24 @@ def _get_classifier_artifacts(fitted_estimator, prefix, X, y_true, sample_weight
 
     def plot_confusion_matrix(*args, **kwargs):
         import matplotlib
+        import matplotlib.pyplot as plt
 
         class_labels = _get_class_labels_from_estimator(fitted_estimator)
         if class_labels is None:
             class_labels = set(y_true)
 
         with matplotlib.rc_context(
             {
-                "figure.dpi": 288,
-                "figure.figsize": [6.0, 4.0],
```

Comment:
changing figure.figsize here doesn't do anything since the inner element inside the figure frame is what sets the size of the rendering (the 'ax' element)